docs/src/manual/gettingstarted.md

```diff
@@ -29,14 +29,10 @@ mc = MC(m, beta=0.35, sweeps=1000, thermalization=1000);
 run!(mc, verbose=false);
 
 # analyze results
-observables(mc) # what observables do exist for that simulation?
-m = mc.obs["m"] # magnetization
+obs = measurements(mc)[:ME] # what measurements exist for that simulation?
```

Comment:
This will become `obs = measurements(mc)` once #41 is merged. May also want to change `obs` to something else